requirements_test.txt

```diff
@@ -12,11 +12,14 @@ meshio<5.4.0
 nest_asyncio<1.5.9
 numpydoc==1.6.0
 param<2.1.0
+PyQt6!=6.6.1,<6.7.0
+PyQt6-Qt6!=6.6.1,<6.7.0
 pytest<7.5.0
 pytest-cov<4.2.0
 pytest-memprof<0.3.0
 pytest-xdist<3.4.0
 pytest_pyvista==0.1.8
+qtpy<2.5.0
```

Comment:
We do not test Qt dependencies in PyVista. This was a part of the entire motivation for making PyVistaQt a separate repository